### PR TITLE
homework array, set and dictionary

### PR DESCRIPTION
**Pull request by KozyrDenis**

### CODE REVIEW

File: 04_CollectionTypes.playground/Contents.swift [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import UIKit
+
+/*
+ MARK: - 🏠 Домашнее задание.
+ 
+ TODO: - 🧑‍💻 Выполните каждый из пунктов для закрепления материала. Решение пишите на следующей строке после задания:
+ // - проинициализируйте два пустых массива foodArray and drinkArray разными способами
+ */
```
Comment (AntonPastrevich) [inline 04_CollectionTypes.playground/Contents.swift:8]: should not be commented, bellow how it should be.
1) проинициализируйте два пустых массива foodArray and drinkArray разными способами
`var foodArray: [String] = []`
2) присвойте первому массиву стринг значения - названия продуктов (минимум 5)
`foodArray = ["burger", "sandwich", "frenchfries", "chocolate", "cheese", "sausage"]`

File: 04_CollectionTypes.playground/Contents.swift [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+//взял в кавычки так как выдавало ошибку
+
+ // - присвойте первому массиву стринг значения - названия продуктов (минимум 5)
+var foodArray: [String] = ["burger", "sandwich", "frenchfries", "chocolate", "cheese", "sausage"]
+
+ // - присвойте второму массиву названия напитков (минимум 5)
+var drinkArray: [String] = ["milk", "cola" , "coffee", "beer" , "water", "juice"]
+
+ // - добавьте в конец foodArray еще одно значение
+foodArray.insert("butter", at: 6)
```
Comment (AntonPastrevich) [inline 04_CollectionTypes.playground/Contents.swift]: What if `foodArray` has more then 5 or even less then 5 items? 
When you insert at: 6 where 6 is an index u must be careful, if `foodArray` has 4 items and u will try to insert at 6 app will crash. 
check pls how `.append()` works. 
or if u would like to use `.insert` u have to use not 6 but `drinkArray.count` to guarantee that u won't have out if range crash.
`drinkArray.insert("butter", at: drinkArray.count)`

File: 04_CollectionTypes.playground/Contents.swift [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+ // - добавьте в середину drinkArray еще одно значение
+drinkArray.insert("coconut milk", at: 3)
```
Comment (AntonPastrevich) [inline 04_CollectionTypes.playground/Contents.swift]: here as well 
what if drinkArray has not 6 items but 8? 
3 won't be a middle index, u need to calculate middle index

File: 04_CollectionTypes.playground/Contents.swift [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+ 
+ // - создайте третий массив shoppingList, который должен включать в себя все значения из foodArray and drinkArray
+ var shoppingList = foodArray + drinkArray
+ 
+ // - распечатайте в консоль название продуктов и напитков из shoppingList и общее количество продуктов в shoppingList.
+print(shoppingList)
+print(shoppingList.count)
+ // - удалите первое и последнее значение, значение по индексу 2 из shoppingList
+shoppingList.remove(at:0)
+shoppingList.remove(at:12)
```
Comment (AntonPastrevich) [inline 04_CollectionTypes.playground/Contents.swift:34]: what if shoppingList has more then 12 items or less ? 
your code must be independent of what u have, if array has 4 item ore 12... if condition is to add item in the middle u need to calculate this middle index

File: 04_CollectionTypes.playground/Contents.swift [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+ // - примените к вышеобъявленным сетами операции symmetricDifference, union, intersection, substracting, isSubset(of:), isSuperset(of:)
+firstSet.symmetricDifference(secondSet)
+firstSet.union(secondSet)
+firstSet.intersection(secondSet)
+firstSet.subtracting(secondSet)
+firstSet.isSubset(of:secondSet)
+firstSet.isSuperset(of:secondSet)
+
+
+// MARK: - 🏠 Готовим ужин.
```
Comment (AntonPastrevich) [inline 04_CollectionTypes.playground/Contents.swift:62]: what about this task?

**Review by AntonPastrevich — CHANGES_REQUESTED** [review]

File: 04_CollectionTypes.playground/Contents.swift (hunk 5 above)
Comment (KozyrDenis) [reply 04_CollectionTypes.playground/Contents.swift:62]: Что касается задач со звездочкой, пока стараюсь не думать над ними, боюсь, что заклинит снова

Comment (AntonPastrevich) [reply 04_CollectionTypes.playground/Contents.swift:62]: так она без звездочки

**Pull request merged**